mobile/src/main/java/ch/epfl/sdp/mobile/application/authentication/AuthenticatedUser.kt

```diff
@@ -91,4 +93,6 @@ class AuthenticatedUser(
           .whereArrayContains("followers", user.uid)
           .asFlow<ProfileDocument>()
           .map { it.mapNotNull { doc -> doc?.toProfile(this) } }
+
+  val pastGames: Flow<List<ChessMatch>> = emptyFlow()
```

Comment:
The `ChessFacade` is responsible to get the chess games for any specific, so you should use its methods instead.